server/handlers/captureHandler.js

```diff
@@ -94,7 +95,28 @@ const captureHandlerPost = async function (req, res) {
   }
 };
 
+const captureHandlerPatch = async function (req, res) {
+  const { capture_id } = req.params;
```

Comment:
we may need to add validation here to check the following parameters are not part of the patch request.
`id`, `lat`, `lon`, `location`, `created_at`.

The idea is that `patch` only supports updating an existing capture by associating it to a specific `tree_id` or `status` or other columns. `id`, and `location` details are not modifiable once set on creation (during POST request).

You could look into the validation library named `joi`  (used in treeHandler.js) to understand how validation is done using a schema if you haven't used that library before.